src/content/docs/release-notes/mobile-release-notes/android-release-notes/android-720.mdx

```diff
@@ -0,0 +1,37 @@
+---
+subject: Android agent
+releaseDate: '2023-11-11'
+version: 7.2.0
+downloadLink: 'https://download.newrelic.com/android_agent/ant/NewRelic_Android_Agent_7.2.0.zip'
+---
+
+## New
+
+### Adds configurable header instrumentation to network requests
+
+The agent will now produce network event attributes if it detects request header values
+that are passed into the agent when started. For more information, see the [Mobile documentation]().
+
+### Adds GraphQL attributes to network requests
+
+If the agent detects headers from Apollo GraphQL servers, it will add additional attributes to the
+network request event. For more information, see the [Mobile documentation]().
+
+* Exclude instrumentation by package name
```

Comment:
Do you mean to use ###Exclude instrumentation by package name here? rather than *?